<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WaitingRoom</class>
 <widget class="QDialog" name="WaitingRoom">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Waiting Room</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>600</height>
    </rect>
   </property>
   <widget class="QLabel" name="labelTitle">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>120</y>
      <width>741</width>
      <height>151</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Jazz Jackrabbit 2</family>
      <pointsize>36</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Conectando...</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelGameName">
    <property name="geometry">
     <rect>
      <x>615</x>
      <y>240</y>
      <width>175</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Jazz Jackrabbit 2</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string></string>
    </property>
   </widget>
   <widget class="QLabel" name="labelPlayers">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>360</y>
      <width>175</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Jazz Jackrabbit 2</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Jugadores:</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelSlash">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>420</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Jazz Jackrabbit 2</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>/</string>
    </property>
   </widget>
   <widget class="QLabel" name="numPlayers">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>420</y>
      <width>40</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Jazz Jackrabbit 2</family>
      <pointsize>20</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="maxPlayers">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>420</y>
      <width>40</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Jazz Jackrabbit 2</family>
      <pointsize>20</pointsize>
     </font>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>0</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="enabled">
    <bool>false</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
